scripts/pr-command.sh

```
@@ -3,4 +3,8 @@ set -ex
 SLASH_COMMAND=$1
 echo "Running pull request slash command: $SLASH_COMMAND"
 
-echo "Hello world
+cd $(dirname $0)/..
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)